LadderManager.cpp

```diff
@@ -1043,7 +1077,7 @@ void LadderManager::RunLadderManager()
 		while (Matchups->GetNextMatchup(NextMatch))
 		{
 			ResultType result = ResultType::InitializationError;
-			std::cout << "Starting " << NextMatch.Agent1.Name << " vs " << NextMatch.Agent2.Name << " on " << NextMatch.Map << " \n";
+			std::cout << "\nStarting " << NextMatch.Agent1.BotName << " vs " << NextMatch.Agent2.BotName << " on " << NextMatch.Map << " \n";
```

Comment:
Better to use std::endl instead of \n.

Reply:
also,  probably best to avoid using \n at the start of the line, one line break should be fine

Reply:
There is no advantage except that it is shorter :)
However \n is considered platform specific.